RevenueCatUI/Templates/Template5View.swift

```diff
@@ -82,6 +82,8 @@ struct Template5View: TemplateViewType {
                     .scrollableIfNecessary()
             }
 
+            Spacer()
```

Comment:
Just curious but how does this look on an iPad in landscape? Just wondering whether the main content will be moved too much to the top, since I think we might want it centered within the available space?

Reply:
iPad landcape doesn't have "vertical height condensed", so this layout isn't used.